app/(with-main-header)/mypage/_components/AlbatalkCard.tsx

```diff
@@ -0,0 +1,87 @@
+'use client';
+import CommentIcon from '@/public/icons/comment.svg';
+import LikeIcon from '@/public/icons/like.svg';
+import Image from 'next/image';
+import Link from 'next/link';
+import { Writer } from '@/types/albatalk';
+import { formatDate } from '@/utils/dateFormatter';
+
+interface AlbatalkCardProps {
+  title: string;
+  content: string;
+  writer: Writer;
+  createdAt: string;
+  commentCount: number;
+  likeCount: number;
+  talkId: number;
+}
+
+const AlbatalkCard = ({
+  title,
+  content,
+  writer,
+  createdAt,
+  commentCount,
+  likeCount,
+  talkId,
+}: AlbatalkCardProps) => {
+  return (
+    <div className="relative w-full h-[210px] lg:h-[280px] p-6 border rounded-2xl">
+      <Link href={`/albatalk/${talkId}`}>
+        <div className="flex flex-col h-[162px] lg:h-[232px] gap-6">
+          <div className="flex lg:h-44 flex-col gap-2">
+            <div className="max-w-80 text-black-400 font-semibold text-lg overflow-hidden lg:text-2lg">
+              {title}
+            </div>
+            <div className="max-w-80 h-[80px] text-gray-500 font-regular text-md overflow-hidden lg:text-lg">
+              {content}
+            </div>
+          </div>
+          <div className="flex">
+            <div className="flex w-full justify-between items-center">
+              <div className="flex gap-4 items-center">
+                <div className="flex gap-2 items-center">
+                  <div className="w-6 h-6 rounded-3xl relative">
+                    <Image
+                      src={
+                        writer && writer.imageUrl
+                          ? writer.imageUrl
+                          : '/icons/profile.svg'
+                      }
```

Comment:
```suggestion
                      src={writer?.imageUrl || '/icons/profile.svg'}
```
💊 이렇게 간단하게 표현 할 수 있을 것 같아요!

Reply:
우왁 감사합니다!